packages/astro/src/default/components/TutorialContent.astro

```diff
@@ -38,4 +38,11 @@ const { Markdown, editPageLink, prev, next } = lesson;
       {next && <NavCard lesson={next} type="next" />}
     </div>
   </div>
+
+  <p class="mt-6 font-size-3.5 text-tk-elements-webcontainersLink-textColor">
+    {lesson.data.i18n!.webcontainerLinkText}
+    <a class="underline hover:text-tk-elements-webcontainersLink-textColorHover" href="https://webcontainers.io/"
+      >webcontainers.io</a
```

Comment:
Shouldn't this say `WebContainer`? Powered by webcontainers.io reads weird to me.

Reply:
Yep I agree. This should be WebContainer instead.

Reply:
`Powered by Webcontainer` or in plural `Powered by Webcontainers`?

Reply:
And as there's no visible URL now, should be make the whole text a link?

<img src="https://github.com/user-attachments/assets/6815a1ad-839b-4aab-a588-20087c39e9dd" width="300" />

Reply:
Good questions! 

I don't think it should be plural as it refers to the technology.

I would probably make the all text the link personally.

Reply:
Now using `Powered by Webcontainer`. Whole text is included in the link element.